templates/analytics/remote_server_support.html

```diff
@@ -83,7 +83,7 @@ <h4>On 100% sponsored Zulip Community plan.</h4>
             <div>
                 {% with %}
                     {% set support_data = remote_realms_support_data %}
-                    {% set get_discount = get_discount %}
+                    {% set get_plan_type_name = get_plan_type_name %}
```

Comment:
`get_discount` was already removed from the context for the remote support view in commit e8500fbdb0.